ingestion-beam/src/test/java/com/mozilla/telemetry/integration/BigQueryIntegrationTest.java

```diff
@@ -450,6 +451,21 @@ public void canLoadDecodedFormat() throws Exception {
             .build()),
         String.join("\n",
             stringValuesQueryWithRetries("SELECT TO_JSON_STRING(t) FROM " + tableSpec + " AS t")));
+
+    String fileOutput = outputPath + "/out";
+    PipelineResult readResult = Sink.run(new String[] { "--inputType=bigquery_table",
+        "--input=" + fullyQualifiedTableSpec, "--project=" + projectId, "--bqReadMethod=storageapi",
+        "--outputFileCompression=UNCOMPRESSED", "--bqSelectedFields=payload",
+        "--bqRowRestriction=CAST(submission_timestamp AS DATE)"
+            + " BETWEEN CAST('2020-01-10' AS DATE) AND CAST('2020-01-14' AS DATE)",
```

Comment:
The storage API is quite picky about how the row restriction is expressed. It looks like the explicit cast syntax is necessary rather than the simpler `DATE(expr)` syntax.

Reply:
I could have sworn that the spark connector worked when I used date literals (e.g. both `submission_date = DATE '2019-01-01'` and `submission_date = '2019-01-01'` worked), even if `CAST(submission_timestamp AS DATE)` is required

Reply:
It looks like you're right. I tested using `DATE('2019-01-01')`, which didn't work, but looks like the even simpler string literal form does work for the date literals.